src/main/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicator.java

```diff
@@ -80,13 +104,12 @@
  * @since 1.0
  */
 public class AverPTZCommunicator extends UDPCommunicator implements Controller, Monitorable {
```

Comment:
I see the methods are missing the javadocs, please add `@param` `@return` `@throws` etc.